Readme.md

```diff
@@ -31,6 +31,75 @@ Place your template files (.cshtml or .liquid) to the *Views/VueComponents* fold
 In these shapes you can use any format you want (e.g. JSX templates) and reference their id in your Vue.js component JavaScript code.
 
 
+## Using Vue.js Single File Components
+
+The module identifies Single File Components in the _Assets/Scripts/VueComponents_ directory and harvests them as shapes. They have a custom _.vue_ file renderer that displays the content of the `<template>` element after applying localization for the custom `[[ ... ]]` expression that calls `IStringLocalizer`. Besides that it's pure Vue, yet you can still make use of shape overriding if needed.
+
+Before you start, add this to your project file so the _.vue_ files are recognized as embedded views:
+```xml
+  <ItemGroup>

```

Comment:
Can we put it to a `.target` file and reference that? I think that'd help if it becomes a .nuget package. @Piedone do you have any comments on this?

Reply:
I think we actually need a `.props` file, right?
Also I've moved the quoted paragraph up to the _Prerequisites_ section to future proof the setup workflow.

Reply:
Yes.